x-pack/test/functional/es_archives/packetbeat/default/mappings.json

```diff
@@ -3160,7 +3160,7 @@
             "dns.question.name",
             "dns.question.type",
             "dns.question.class",
-            "dns.question.etld_plus_one",
+            "dns.question.registered_domain",
```

Comment:
That worked, I always tried to modify JSON and I failed each time. Do you have a technique?

Reply:
I still modified the JSON.

I ran:

```ts
node ../scripts/es_archiver edit packetbeat/dns
```

And then edited it, closed it and then re-pushed.